joomla/joomla-cms
# Fixing #__assets table

Remove unused com_massmail and added missing assets.

## **Testing instructions**

1. Install Joomla! as usual
2. Check if everything works fine.
